### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by sagargse**
created avatar component

### CODE REVIEW

File: documentation.html [hunk 1]
```diff
@@ -64,7 +64,10 @@ <h2>Components</h2>
                 <div id="alerts">
                     <div class="components-intro">
                         <h1>Alerts</h1>
-                        <p>Alerts are used to attract user's attention for important information without interrupting the user's flow.<br>Alerts are available in five types-<span style="color: rgba(255, 0, 0, 0.767);">success alert, error alert, warning alert, primary alert, secondary alert.</span></p>            
+                        <p>Alerts are used to attract user's attention for important information without interrupting
+                            the user's flow.<br>Alerts are available in five types-<span
+                                style="color: rgba(255, 0, 0, 0.767);">success alert, error alert, warning alert,
+                                primary alert, secondary alert.</span></p>
```
Comment (yashpatel1797) [inline documentation.html:70]: Avoid the use of inline CSS.

**Review by yashpatel1797 — COMMENTED** [review]
code looks good to me.
just try to avoid using inline CSS.

File: documentation.html (hunk 1 above)
Comment (sagargse) [reply documentation.html:70]: ok i will resolve that

File: cssfiles/nav-bar.css [hunk 2]
```diff
@@ -4,7 +4,9 @@
     justify-content: space-between;
     box-shadow: rgba(33, 35, 38, 0.1) 0px 10px 10px -10px;
 }
```
Comment (Shubhamkr2610) [inline cssfiles/nav-bar.css:6]: use color name with variable. it will help in code readability

File: documentation.html [hunk 3]
```diff
@@ -111,9 +114,62 @@ <h2>JavaScript</h2>
...
+                            src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=one-dark&wt=none&l=auto&width=680&ds=true&dsyoff=0px&dsblur=0px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522flex-items%2522%253E%250A%2509%253Cdiv%2520class%253D%2522avatar%2520big-avatar%2520text-avatar%2522%253ESR%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522flex-items%2522%253E%250A%2509%253Cdiv%2520class%253D%2522avatar%2520md-avatar%2520text-avatar%2522%253ESR%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522flex-items%2522%253E%250A%2509%253Cdiv%2520class%253D%2522avatar%2520sm-avatar%2520text-avatar%2522%253ESR%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522flex-items%2522%253E%250A%2509%253Cdiv%2520class%253D%2522avatar%2520exsm-avatar%2520text-avatar%2522%253ESR%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%2520">
+                        </iframe>
+                    </div>
+                </div>
+                <hr>
+            </section>
         </main>
-
         <footer class="bottom-footer-bar">
```
Comment (arjun-computer-geek) [inline documentation.html:173]: Don't use left, right, bottom  as name as class. It

File: cssfiles/nav-bar.css [hunk 4]
```diff
@@ -4,7 +4,9 @@
...
-
+.logo>h1{
+    color:var(--primary);
+}
```
Comment (Shubhamkr2610) [inline cssfiles/nav-bar.css:9]: give class to h1 then use css to h1. else it will overwrite

**Review by arjun-computer-geek — COMMENTED** [review]
use meaningful title according to the documents

**Pull request closed without merging**